submit/submit.go

```diff
@@ -113,7 +113,7 @@ func getTmp(cli *cliargs) string {
 	tmp := fmt.Sprintf(`# thanks Hao
 export TMPDIR="$(mktemp -d -p %s)"
 cleanup() { echo "batchit: deleting temp dir ${TMPDIR}"; umount -l /tmp/; rm -rf ${TMPDIR}; }
-trap cleanup EXIT
+trap "cleanup_volume EXIT; cleanup;" EXIT
```

Comment:
got an extra EXIT in this line?

Reply:
cleanup_volume() takes the signal as a parameter

Reply:
got it. can you verify this works on your set-up and then i'll merge?

Reply:
confirmed, successfully ran latest code update with and without the `--ebs` option